pkg/util/clusteragent/clusteragent.go

```diff
@@ -93,6 +93,13 @@ func (c *DCAClient) init() error {
 	c.clusterAgentAPIClient = util.GetClient(false)
 	c.clusterAgentAPIClient.Timeout = 2 * time.Second
 
+	// Validate the cluster-agent client by checking the version
+	ver, err := c.GetVersion()
```

Comment:
A good TODO could be to store the ver in the DCAClient so we can do some version checking in the future WDYT ?